docs-md/developer-guide/ksqldb-reference/show-topics.md

```diff
@@ -24,10 +24,36 @@ configured to connect to (default setting for `bootstrap.servers`:
 `localhost:9092`). SHOW TOPICS EXTENDED also displays consumer groups
 and their active consumer counts.
 
+SHOW TOPICS does not display topics considered internal, such as:
+* KSQL internal topics, like the KSQL command topic
+* Topics found in the `ksql.internal.hidden.topics` configuration
+
+SHOW ALL TOPICS lists all topics, including those considered to be
+internal or found in the `ksql.internal.hidden.topics` configuration.
+

```

Comment:
I find the 'internal.hidden' part of the config confusing.  To me, the `internal` part is referring to KSQL's internal topics, (e.g. command topic, changelog + repartition topics, etc). But these are not controlled by the `ksql.internal.hidden.topics`!  

`ksql.internal.hidden.topics` is there to allow users to configure additional hidden topics. Hence I think `ksql.hidden.topics` would be a better name and the text above would then be:

```
`SHOW TOPICS` does not display hidden topics by default, such as:
* KSQL internal topics, like the KSQL command topic or changelog & repartition topics.
* Topics that match any pattern in the `ksql.hidden.topics` configuration.

`SHOW ALL TOPICS` lists all topics, including hidden topics.
```

IMHO this is a lot clearer.